Jenkinsfile

```diff
@@ -0,0 +1,46 @@
+pipeline {
+    libraries {
+        lib('utils')
+    }
+    agent {
+        label 'mac'
+    }
+    stages {
+        stage('init-submodule') {
+            steps {
+                script {
+                    utils.run("git submodule update --init")
+                }
+            }
+        }
+        stage('build') {
+            steps {
+                script {
+                    utils.run("""
+                        xcodebuild -target Sparkle -configuration Release CONFIGURATION_BUILD_DIR=binaries build
+                        xcodebuild -target BinaryDelta -configuration Release CONFIGURATION_BUILD_DIR=binaries build
+                        xcodebuild -target generate_keys -configuration Release CONFIGURATION_BUILD_DIR=binaries build
+                        xcodebuild -target sign_update -configuration Release CONFIGURATION_BUILD_DIR=binaries build
+                        cp bin/old_dsa_scripts/sign_update binaries/sign_update_dsa
+                    """)
+                }
+            }
+        }
+        stage('pack-binaries') {
```

Comment:
`dist`

Reply:
Resolved in 4c588204cd9515bfd7e89a9e360e4bb5b820cfdc.

Reply:
did you want `build` or `pack-binaries` to become `dist`? I guess I misunderstood and changed `build` to `dist`.

Reply:
`pack-binaries` to `dist`

Reply:
Resolved in f0f5f2a9a749a1d4820db5ac379efcc61d1e359c.